Apps/ValidationTests/Scripts/validation_native.js

```diff
@@ -3,7 +3,7 @@ var currentScene;
 var config;
 var justOnce;
 var threshold = 25;
-var errorRatio = 2.5;
+var errorRatio = 8.0;
```

Comment:
Is this higher than what Babylon.js uses now?

Reply:
Yes! 2.5 bjs 8 here. Mostly because text is rendered in some PG.

Reply:
I'm assuming you mean text is not being rendered because we don't have text rendering yet?

Reply:
Yes! That's correct.

Reply:
We discussed this offline and the plan is to update this value only for the tests that have text and put a comment somewhere that explains why (or, at a minimum, make sure the commit message says why).